examples/experimental/amazonlinux-2023.yaml

```diff
@@ -0,0 +1,15 @@
+# This template requires Lima v0.17.0 or later and requires macOS >=13.0
+mountType: "virtiofs"
```

Comment:
I don't think the mountType should be in here.

Reply:
Both other mount types are broken on Amazon Linux, so the only alternative is removing default mounts...

Reply:
Given that the PR needs another change anyways, I think it would be good to add a comment here that `reverse-sshfs`, and `9p` are not supported by amazonlinux, so people know not to bother trying.

Reply:
fixed